src/main/java/ru/eristarion/homework/cowoker/Coworker.java

```diff
@@ -33,5 +33,6 @@ public void printInfo() {
         System.out.printf("Имя: %s\nФамилия: %s\nОтчество: %s\nEmail: %s\nЗарплата: %d\nВозраст: %d", name, surName, patronymic, email, salary, age);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.